### DISCUSSION TITLE
Feature request: wrap text in Table of Contents

### DISCUSSION THREAD

**Discussion by jmkorhonen** [zim-desktop-wiki/zim-desktop-wiki#2071] [Ideas]
Hi and many, many thanks for this exceptional personal information system. 

I use the Zim to write the drafts of my books and papers, and for that purpose, I've found the Table of Contents very useful. I use it to outline the argument I try to make by writing the essential point of each paragraph or section, and then formatting this to heading. The point now shows up in the ToC, and seeing the overall structure of the argument at a glance is very helpful. 

However, I need to keep the points short because the ToC does not wrap the text. Using ToC as an outliner tool would therefore be much improved if the ToC wrapped the text instead of overflowing it. Might such a feature be feasible to implement at some point?

In any case, thanks again for all the great work!

**Comment by quesada** [comment]
+1
